release/README.md

```diff
@@ -200,6 +200,15 @@ next 24 hours.
 (Optional) Create a PR to update the `google-cloud-cpp` package in
 [Conan](https://conan.io). These PRs are more involved.
 
+This package manager uses CMake, but ignores the library and target definitions
+exported by our CMake files. It reimplements the library definitions and
+dependencies in Python code. The recipes do this for every package (gRPC,
+Protobuf, Boost, etc.), not just for `google-cloud-cpp`. For most packages,
+where there are only a handful of libraries, this is trivial. `google-cloud-cpp`
+has about 240 libraries (including the `*_protos` libraries). So, we use a
```

Comment:
Looks good.

My only final suggestion would be, _"But `google-cloud-cpp` has ..., so we use ..."_.

Reply:
Done.